libs/serve-stats.js

```diff
@@ -1,19 +1,15 @@
 const axios = require('axios')
 const { send } = require('micro')
 
-module.exports = async function (req, res) {
-  const [
-    githubRateLimit
-  ] = await Promise.all([
-    getGithubRateLimit()
-  ])
+module.exports = async (req, res) => {
+  const [githubRateLimit] = await Promise.all([getGithubRateLimit()])
```

Comment:
I thought there's more to be added here, but right now the Prettier version are better for sure 😅